File: Day_7/JS/TaskSeven.js
let number = 1;
let sum_first_pair = 0;
let sum_second_pair = 0;
let pair = 0

while( number <= 10 ){
	
	if(number % 4 == 0){
		pair++
		let constant_number = number
		let count = 0;
		while (count < 5){
			if( pair == 1 )
				sum_first_pair += number;
			else if( pair == 2 )
				sum_second_pair += number;
			number *= constant_number;
			count++;
		}
		number = constant_number;
	}
	number++;
	

}	
console.log(sum_first_pair);
console.log(sum_second_pair);

File: Day_7/Java/TaskSeven.java
public class TaskSeven{
	public static void main(String [] args){
		int parenthesis = 0;
		int constantNumber = 0;
		int sumFirstPair = 0;
		int sumSecondPair = 0;
		int pair = 0;
		for(int number = 1; number <= 10; number++){
			if( number % 4 == 0 ){
				pair++;
				constantNumber = number;
				if (parenthesis < 1){
					parenthesis ++;
				}
			for(int count = 1; count <= 5; count++){
				if( pair == 1 )
					sumFirstPair += number;
				else if( pair == 2 )
					sumSecondPair += number;

				number *= constantNumber;}
			number = constantNumber;	
			parenthesis++;
			
			}

		
		}	
		System.out.printf("(%d %d)", sumFirstPair, sumSecondPair);

	}

}